src/code/init.js

```diff
@@ -48,6 +48,14 @@ window.plugin.wasabee.init = function() {
     drawAgents(Wasabee._selectedOp);
   });
 
+  window.addHook("portalDetailsUpdated", e => {
+    listenForPortalDetails({
+      success: true,
+      guid: e.guid,
+      details: e.portalDetails
+    });
+  });
+
```

Comment:
in earlier tests, having this always-on was SLOW. is it better now?

Reply:
it is only for portalDetailsUpdated, and this hook is only triggered when the sidebar updates (by renderPortailDetails), it does not happen often

I didn't test against `portalDetailsLoaded`, I woul'd rather use something that wait for everything to be load and pull the info from iitc, but it might be slow to